src/components/DatePicker/index.js

```diff
@@ -19,8 +19,6 @@ class DatePicker extends React.Component {
 
         this.setDate = this.setDate.bind(this);
         this.showDatepicker = this.showDatepicker.bind(this);
-
-        this.defaultValue = props.defaultValue ? moment(props.defaultValue).format(CONST.DATE.MOMENT_FORMAT_STRING) : '';
```

Comment:
@hannojg Are we good to remove `defaultValue`? `defaultValue` prop is being passed to DatePicker from different components.

Reply:
Yeah, i tested it. Its okay, because the DatePicker is only used in Forms. Form.js will copy the datepicker and apply the correct values to it.

Reply:
However, i'd definitely appreciate a double testing by you of the affected components!